### PR TITLE
Adding flag to set operator default reconcile-period.

### PR DESCRIPTION
**Pull request by shawn-hurley**
* Updating to operator-sdk.

### CODE REVIEW

**Comment by djzager** [comment]
So this update to the command line options, makes it so `ansible-operator --reconcile-period=5m` would make this a default for each of the CRs in my watches.yaml unless I specifically override it?

**Comment by fabianvf** [comment]
needs rebase, other than that no opposition.

**Comment by shawn-hurley** [comment]
@fabianvf @dymurray @djzager Can you guys look at this?

File: cmd/manager/main.go [hunk 1]
```diff
@@ -11,23 +10,31 @@ import (
...
 }
 
 func main() {
-	flag.Parse()
+	pflag.Parse()
 	logf.SetLogger(logf.ZapLogger(false))
 
+	d, err := time.ParseDuration(*defaultReconcilePeriod)
+	if err != nil {
+		log.Fatalf("failed to get watch namespace: %v", err)
```
Comment (djzager) [inline cmd/manager/main.go]: This message doesn't look right. Should it be "failed to get reconcile-period"?

**Pull request merged**